ibtk/include/ibtk/HierarchyMathOps.h

```diff
@@ -498,6 +498,37 @@ class HierarchyMathOps : public SAMRAI::tbox::DescribedClass
              int dst_depth = 0,
              int src2_depth = 0);
 
+    /*!
+     * \brief Compute the cell-centered divergence of product of two side-centered normal
+     * vectors field using centered differences.
+     *
+     * Sets dst = alpha div (coef1 * src1) + beta src2.
+     *
+     * Compute the divergence of product of two side-centered vector fields using centered differences.
+     * When specified, coarse values on each coarse-fine interface are
+     * synchronized prior to performing the differencing.
+     *
+     * \see setPatchHierarchy
+     * \see resetLevels
+     */
+    void div(int dst_idx,
+             SAMRAI::tbox::Pointer<SAMRAI::pdat::CellVariable<NDIM, double> > dst_var,
+             double alpha,
+             int src1_idx,
+             SAMRAI::tbox::Pointer<SAMRAI::pdat::SideVariable<NDIM, double> > src1_var,
```

Comment:
Two comments:

1. The convention for most of these operators is that "coefficients" come before the "variable" that they are hitting. Can we stick with that convention here? (This is related to my question above, about whether `coef1` should be thought of as a "problem coefficient" or a "variable".)
2. Can we merge together `alpha` and `coef1` so that there is just one coefficient per variable?